tests/Framework/AssertTest.php

```diff
@@ -3177,6 +3177,43 @@ public function testAssertStringNotMatchesFormatFile()
         $this->assertStringNotMatchesFormatFile($this->filesDirectory . 'expectedFileFormat.txt', "FOO\n");
     }
 
+    public function testAssertResourceWithTypeForMemoryStream()
+    {
+        // GIVEN
```

Comment:
I didn't saw this approach of setting up tests in this codebase,
and test methods are extra small, I don't find any benefit of those comments tbh

Reply:
Sorry, it became my habit.
I agree, comments removed.